roles/identity-management/manage-rh-sso-identities/tasks/access-token.yml

```diff
@@ -9,5 +9,6 @@
       password: "{{ rh_sso_admin_pass }}"
       grant_type: "password"
       client_id: "admin-cli"
+      validate_certs: "{{ rh_sso_ssl_cert_validation | default(omit) }}"
```

Comment:
wrong indentation ... need to pull that back 2 spaces as it otherwise will be part of the `body` :-)